src/Redux/Reducers/quizSlice.tsx

```diff
@@ -61,9 +81,19 @@ const quizSlice = createSlice({
       .addCase(
         getAllQuizes.fulfilled,
         (state, action: PayloadAction<QuizState>) => {
-          state.quizes = action.payload.quizes
+          state.quizes = action.payload.quizes;
         }
-      );
+      )
+      .addCase(getAllQuizes.rejected, (state, action) => {
+        console.log(action.payload);
```

Comment:
you can find another way of showing rejected vals.. instead of console log.

Reply:
I am yet to add a toast to show errors and success. Thanks for pointing it out!